### ISSUE TITLE
Twitch streams being blocked by brave shield

### ISSUE THREAD

**Issue by indieindie7** [brave/browser-laptop#12160]
### Description
While brave shield is on, streams on twitch.tv are not able to be watched, i tried many options, and it appears to be just that. it just keeps loading or nothing happens

### Steps to Reproduce
  1. open twitch.tv
  2.turn on brave shield
  3. go to any channel
  4. :/

**Actual result:**
Stream will not load

**Expected result:**
Video should play without ads, twitch ads are horrible they just pop out of nowhere

**Reproduces how often:**
Always

### Brave Version
v0.19.105

### Additional Information
still works on chrome with adblock, so i dont really know whats happening, besides chrome being slow and having big network consumption nothing uncommon

**Comment by eljuno** [comment]
Related to #6872?

**Comment by indieindie7** [comment]
yes, i looked over it, but allowing ads did not make the video stream function properly

**Comment by sudomake** [comment]
This started with a recent Brave update.  Disabling Fingerprinting Protection allows the video to load.

WebGL is the fingerprinting method being blocked and stopping the videos from loading.
